src/mongo_types.ts

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable @typescript-eslint/ban-types */
```

Comment:
This is to permit the `{}` type which means anything not null.

This file does compile to an empty javascript file currently. However I think that [NODE-3132](https://jira.mongodb.org/browse/NODE-3132) will add code that will actually compile to something in this file so its worth leaving in.

Reply:
Leaving this comment open to see if we still even need the ban-types; option to refactor existing types to no longer need to reference {}